internal/server/storage/drivers/driver_nfs.go

```diff
@@ -0,0 +1,125 @@
+package drivers
+
+import (
+	"strings"
+
+	"github.com/lxc/incus/v6/internal/linux"
+	"github.com/lxc/incus/v6/internal/migration"
+	deviceConfig "github.com/lxc/incus/v6/internal/server/device/config"
+	localMigration "github.com/lxc/incus/v6/internal/server/migration"
+	"github.com/lxc/incus/v6/shared/util"
+)
+
+type nfs struct {
+	dir
+}
+
+// Info returns info about the driver and its environment.
+func (n *nfs) Info() Info {
+	return Info{
+		Name:                         "nfs",
+		Version:                      "1",
+		DefaultVMBlockFilesystemSize: deviceConfig.DefaultVMBlockFilesystemSize,
+		OptimizedImages:              false,
+		PreservesInodes:              false,
+		Remote:                       n.isRemote(),
+		VolumeTypes:                  []VolumeType{VolumeTypeBucket, VolumeTypeCustom, VolumeTypeImage, VolumeTypeContainer, VolumeTypeVM},
+		VolumeMultiNode:              n.isRemote(),
+		BlockBacking:                 false,
+		RunningCopyFreeze:            false,
+		DirectIO:                     true,
```

Comment:
Not sure about that one, does NFS handle O_DIRECT?

Reply:
I've left and spawing qemu instances seems to work just fine.